### ISSUE TITLE
Can't insert text into rich text editor using JS

### ISSUE THREAD

**Issue by kkaliannan** [tiff/wysihtml5#515]
I want to insert some HTML (html_string) into the textarea using:

var wysihtml5Editor = editorElement.data("wysihtml5").editor;
wysihtml5Editor.composer.commands.exec("insertHTML", html_string);

However, this only seems to work if I have first clicked on the textarea at least once. Has anyone else seen this? (the JS debugger yields no errors).

**Comment by emiller12** [comment]
Yes I am seeing the exact same problem. Did you ever find a solution?

**Comment by caribesoft** [comment]
I have the same problem with CKEDITOR. Does anyboby found a solution ?
